apps/web/src/constants/permissionConstants.ts

```diff
@@ -1,30 +1,39 @@
-import { RoleName } from "@gc-digital-talent/auth";
+import { ROLE_NAME, RoleName } from "@gc-digital-talent/auth";
 
 const permissionConstants = () => {
```

Comment:
Sure. I guess a function means you can't accidentally modify it? But I think we can avoid doing that. Updated in 7fdea776a49994f01b1093819188ad5dd9814ea4

Reply:
Fair, though if you want to avoid that you can also use the `Readonly` utility.

 - [Documentation](https://www.typescriptlang.org/docs/handbook/utility-types.html#readonlytype)
 - [Example](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBCBGArAXDASgUwIYBNwBsBPAHk1ACcdjpyBLMAcwBoYb6GBtAXQD4eYAvDADeAKBgSYGHLSioO4yUoBEuALb1lTRUomqADvvy1gWMFGU6YXUQF8A3KNEJEAOmmzBMDstBqNYLSEylz2QA)

Reply:
Cool, done 043e5954eeaf9105bb9d67b03ccd61ca907e67ec